vue.config.js

```diff
@@ -3,16 +3,16 @@ const CompressionPlugin = require('compression-webpack-plugin');
 module.exports = {
   devServer: {
     proxy: {
-      "/": {
+      '/': {
         target: process.env.BASE_URL,
         onProxyRes: proxyRes => {
-          if (proxyRes.headers["set-cookie"]) {
+          if (proxyRes.headers['set-cookie']) {
             // Need to remove 'Secure' flag on set-cookie value so browser
             // can create cookie for local development
-            const cookies = proxyRes.headers["set-cookie"].map(cookie =>
-              cookie.replace(/; secure/gi, "")
+            const cookies = proxyRes.headers['set-cookie'].map(cookie =>
+              cookie.replace(/; secure/gi, '')
             );
-            proxyRes.headers["set-cookie"] = cookies;
+            proxyRes.headers['set-cookie'] = cookies;
           }
         }
       }
```

Comment:
Since all that is changed is formatting, I don't think this would be included in this commit.  I am assuming it was flagged by the checkin process? Is that right? If so, it's fine because it's small. :)